package.json

```diff
@@ -97,5 +91,9 @@
     "*.js": ["eslint --fix", "git add"]
   },
   "module": "lib/index.es.js",
-  "pre-commit": "lint:staged"
+  "pre-commit": "lint:staged",
+  "dependencies": {
+    "iotacss": "^1.5.0",
+    "react-iotacss": "^1.3.1"
```

Comment:
Why is this not a `devDependency` (we would also need to remove it from `externals` below if changed here)?

Reply:
Are we assuming that any react application that installs this will also need to use `react-iotacss`?

Reply:
`iotacss` is a dep rather than devDep so /learning can remove its own copy of the `iotacss` dependency. This changes means that when /learning's non-Bits sass files reference iotaCSS, it's getting the files from the copy supplied by Bits.

Same deal with `react-iotacss` - except for React components instead of styles. Some of the `react-iotacss` components are exported via Bits, but if we wanted to just include stuff from react-iotacss then this setting allows that.

Reply:
Does `slash-learning-ui` need `react-iotacss` within its own application? If not I would make it a `devDependency` here & remove from the externals list in rollup config.

Reply:
Not sure. Will test. I know that `iotacss` definitely does have to be listed as a dependency while we're still importing in within the application styles, but will try to remove the `react-iotacss`.

Reply:
@aldreth `react-iotacss` appears to have moved to `devDependencies` just fine.

Reply:
Although it might be worth me getting an updated /learning review app up with this version of Bits to make sure it works ok in prod. Good idea?

Reply:
Created a PR to test this on /learning here: https://github.com/RaspberryPiFoundation/slash-learning-ui/pull/733

Don't need to merge that because we'll be updating /learning to the tagged version of Bits THIS pull request will result in. Only need the review app to see that this change doesn't kill /learning :)

Reply:
@aldreth Circle failed to test the version with react-iotacss in devDependencies: https://circleci.com/gh/RaspberryPiFoundation/slash-learning-ui/2134 - I remember that was why I put it in there in the first place!

Reply:
Ok, fair enough!